index.js

```diff
@@ -1417,6 +1417,24 @@
     return maybe.isNothing ? Left(x) : Right(maybe.value);
   });
 
+  //. ### Function
+
+  //# flip :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. Takes a binary function and two values of any type and returns
+  //. the result of applying the function - with its argument order
+  //. reversed - to the values. `flip` may also be applied to functions
+  //. with an arity greater than two that have been appropriately
+  //. curried.
+  //.
+  //. ```javascript
+  //. > S.flip(S.indexOf, ['a', 'b', 'c', 'd'], 'c')
+  //. S.Just(2)
+  //. ```
```

Comment:
This doesn't demonstrate the usefulness of `flip`, since we could have applied `indexOf` directly. I suggest:

```javascript
> R.map(S.flip(Math.pow)(2), [1, 2, 3, 4, 5])
[1, 4, 9, 16, 25]
```